File: task_03/c/prime.c
#include <stdbool.h>
#include <stdio.h>
int main() {
  int i, j, n;
  printf("enter a positive number: ");
  scanf("%d", &n);
  if (n < 0) {
    printf("not valid");
  } else {
    for (i = 2; i < n + 1; i++) {
      bool isprime = true;
      for (j = 2; j < i ; j++) {
        if (i % j == 0) {
          isprime = false;
          break;
        }
      }
      if (isprime == true) {
        printf("%d\n", i);
      }
    }
  }
}

File: task_03/rust/prime.rs
use std::io;

fn main() {
    let mut input = String::new();

    println!("Enter a positive number :");
    io::stdin().read_line(&mut input).expect("");

    let num: i32 = input.trim().parse().expect("");
    if num <= 0 {
    println!("not valid");
    }else {
    for i in 2 .. num {
    let mut isprime = true;
    for j in 2 .. i {
    if i%j == 0 {
    isprime = false;
    break;
    }}
    if isprime == true {
    println!("{}",i);
    }}}

}


